### PR TITLE
Add draft for hometime

### PR DESCRIPTION
**Pull request by raging-hawk**
Adding Food and Activites

:tada:

### CODE REVIEW

File: raging-hawk-Antrim.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+### Antrim
```
Comment (githubteacher) [inline raging-hawk-Antrim.md:2]: @raging-hawk where is Antrim? ❓

File: raging-hawk-Antrim.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+#### Food :grin:
+  - Millside
+
+#### Things to do 
+  - coasteering
```
Comment (githubteacher) [inline raging-hawk-Antrim.md:8]: @raging-hawk is this related to roller coasters?

File: raging-hawk-Antrim.md (hunk 1 above)
Comment (raging-hawk) [inline raging-hawk-Antrim.md:2]: Ireland
